builtin/providers/google/resource_compute_address.go

```diff
@@ -114,10 +131,10 @@ func resourceComputeAddressDelete(d *schema.ResourceData, meta interface{}) erro
 	// Wait for the operation to complete
 	w := &OperationWaiter{
 		Service: config.clientCompute,
-		Op:      op,
+		Op:		 op,
```

Comment:
The original formatting is `gofmt`-compliant - i.e. spaces were ok here, no need to change this to tabs.